other/materials_designer/create_interface_with_relaxation_ase_emt.ipynb

```diff
@@ -4,436 +4,196 @@
    "cell_type": "markdown",
```

Comment:
<p>Optimize atomic coordinates of the interface using the EMT (Effective Medium Theory) potential and BFGS.</p><p> </p><p> </p><p>To get interface material one can run Create Interface with Min Strain ZSL notebook first. Upon completion, The resulting interface from it will be accessible in the current notebook as explained in Data exchange.</p><p> </p><p> </p><p>PLUS:</p><ul><li>is the comment 2 about - Make sure to select even relevant anymore?</li><li>Cell 1.1 - remove</li><li>Set relaxation parameters in cell 1.1, not 2.1.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/126/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Well, 2. -- you still need and able to select the materials, maybe  here it's already explained with the data exhange link